packages/server/lib/cloud/api/index.ts

```diff
@@ -307,7 +309,10 @@ let preflightResult = {
 
 let recordRoutes = apiRoutes
 
-module.exports = {
+// Potential todos: Refactor to named exports, refactor away from `this.` in exports,
+// move individual exports to their own files & convert this to barrelfile
+
+export default {
```

Comment:
Since being able to export types was fairly important here, the main export had to change. Since several methods on this export reference other exported methods via `this`, this is only an intermediary step. Extracting these methods to their own files is out of scope for this refactor.